packages/xo-web/src/common/store/reducer.js

```diff
@@ -163,4 +163,10 @@ export default {
       },
     }
   ),
+  xoaReleaseChannels: combineActionHandlers(
+    {},
```

Comment:
The default value should probably be `undefined`.

Reply:
This should be an empty array given that `getReleaseChannels` return an  `array`

Reply:
As you wish, `undefined` could have meant that the channels have not been fetched yet, but it's entirely up to you :slightly_smiling_face: